frontend/ui/wikipedia.lua

```diff
@@ -392,12 +392,12 @@ function Wikipedia:createEpub(epub_path, page, lang, with_images, progress_callb
     if with_images then
         -- if no progress_callback (non UI), our fake one will return true
         if #images > 0 then
-            include_images = progress_callback(T(_("Page contains %1 images.\nWould you like to download and include them in epub ?"), #images), true)
+            include_images = progress_callback(T(_("The page contains %1 images.\nWould you like to download and include them in the generated EPUB file?"), #images), true)
             if include_images then
-                use_img_2x = progress_callback(_("Would you like to get slightly higher quality images (but bigger file size) ?"), true)
+                use_img_2x = progress_callback(_("Would you like to use higher quality images? This will result in a bigger file size."), true)
```

Comment:
Note that there are some disagreements about the best way to phrase this. My position is that including "Do you want to" or "would you like to" is the standard, polite way of phrasing questions in English and is not, as such, redundant.

![how-to-access-hidden-merge-option-in-mac-os-x-finder](https://cloud.githubusercontent.com/assets/202757/22465636/f458c39e-e7bc-11e6-98fb-71fc6e65dff0.png)
![r0nph](https://cloud.githubusercontent.com/assets/202757/22465637/f46d228a-e7bc-11e6-9298-f34e57c6fb70.png)

<hr>
A copy of the discussion [here](https://www.transifex.com/houqp/koreader/translate/#nl_NL/koreader/106518020?issue=yes):

@kingu Urgh, I just lost a big comment by accident. I shouldn't compose in the browser The gist of it was that I don't think "do you want" is redundant because we're not writing telegrams but regular English, including politeness markers. "Would you like to X?" is a standard Windows pattern and "Do you want to X?" is a standard Linux and Mac pattern, in each case followed by an explanation in the next sentence or after a line break: http://www.howtogeek.com/wp-content/uploads/2014/10/how-to-access-hidden-merge-option-in-mac-os-x-finder.png
by Frenzie, 15 hours ago
Edit
nb_NO
@Frenzie I think you are adding words, to the detriment of adding words, and not to anyone's benefit.
Having "resolution", could make it short. Though "improved quality" is also included. So is this menu the place to educate people on the correlation?
"Do you want to" is similarly redundant.
You are using the for operator, which you had a problem with unless the whole explanation was there. I think that was a valid concern, that now makes it a bumpy road of a sentence.

How about
"Increase image-quality, but also filesize?
or
"Increase image-quality and filesize"?
by kingu, 16 hours ago
nb_NO
@kingu "Do you want to increase image resolution and file size for improved quality?" works for me.
by Frenzie, 18 hours ago
nb_NO
@Frenzie Yes, that is the tradeoff. "Increase resolution and filesize of images" would work (?)
by kingu, 19 hours ago
nb_NO
@kingu The reason it doesn't work is the same reason that "Increase image quality in return for bigger file size?" doesn't either. You need to say "Increase file size for better image quality?" You do something "negative" in return for something "positive".
by Frenzie, 19 hours ago
nb_NO
@Frenzie Usually for in this sense means 'in return for'.
Include images y/n
Higher quality, bigger images y/n ?

As it is read
"Would you like to include higher quality images?
(there werent any before, or?)
This will increase the file size."
(oh there were, and this will only increase the size of those)
by kingu, 20 hours ago
nb_NO
@kingu That's correct: there aren't any images there to begin with. First the user is asked whether to include images, provided there are images to include. If they answer in the affirmative, they're asked whether to include the default quality or 2x quality.

I'm not entirely sure what you mean by your objection regarding clarification ("context"). Could you please clarify why it's not optimal? It's commonly used, such as in "File X already exists. Do you want to it? Replacing the file will overwrite its content." I would say that trying to stuff too much into a single sentence reduces clarity.
by Frenzie, a day ago
nb_NO
@Frenzie That tells me the images werent there to begin with. Andd then clarifies with context, which isnt optimal.
by kingu, a day ago
nb_NO
@kingu I think you mean "in spite of", not "for". :-P

I'd be more inclined toward something along the lines of "Would you like to include higher quality images? This will increase the file size."
by Frenzie, a day ago
nb_NO ISSUE
How about "Increase image quality for bigger file size?"